### DISCUSSION TITLE
i cannot seem to be able to run meteor client with optifabric it says look for version 0.4.7 so i do so and i dont find a version called optifabric_0.4.7

### DISCUSSION THREAD

**Discussion by JadenMart** [FabricMC/fabric-api#2059] [FAQ] [answered]
i cannot seem to be able to run meteor client with optifabric it says look for version 0.4.7 so i do so and i dont find a version called optifabric_0.4.7       
![image](https://user-images.githubusercontent.com/101213904/157311991-238757bd-0ea6-450f-b872-83e6865f2806.png)

**Comment by xanthian** [comment, marked as answer]
We do not support hacked clients, cheats, or modifications that are intended to give you the upper hand in a server, competitive or otherwise.

**Comment by JadenMart** [comment]
i literally sent this request like 2 months ago i dont hack anymore
